src/fstar/FStar.Main.fs

```diff
@@ -91,7 +91,7 @@ let load_native_tactics () =
     let cmxs_files = (modules_to_load@cmxs_to_load) |> List.map cmxs_file in
     if Options.debug_any () then
       Util.print1 "Will try to load cmxs files: %s\n" (String.concat ", " cmxs_files);
-    if not (Options.no_load_fstartaclib ()) && not (FStar.Platform.system = FStar.Platform.Windows) then
+    if not (Options.no_load_fstartaclib ()) then //&& not (FStar.Platform.system = FStar.Platform.Windows) then
```

Comment:
@R1kM , could we just remove the commented code?

Reply:
I guess it would to keep it as is for steel merge

Reply:
Sure, I'll need to merge master back into steel anyway